### PR TITLE
Create chmod.md

### PR DESCRIPTION
**Pull request by ls6758**

### CODE REVIEW

File: docs/chmod.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+```
+Verbose, which shows changed objects (and doesn’t show unchanged objects)
+
+##Modes
+Modes specify the read, write, and execute access, respectively for the three classes of user, group, and others. This is represented as a numerical value, typically either 3 or 4 characters in length. A mode of 755 is typical, and will make a file readable and executable by all three user groups, and writable by the owner only.
+
+##Output
+This command does not output a message. The ls or stat command would need to be used in order to show that the changes were made and saved correctly. To check if the change of the root folder permissions worked as intended in our Omeka setup, the following command would be run: 
+```
+ls -l /var/www/html/omeka/
```
Comment (jmf-nyu) [inline docs/chmod.md:29]: Well done!

File: docs/chmod.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+-R
+```
+Recursive, which will include objects in subdirectories. This option must be input as a capital R to work as intended.
+```
+-v
+```
+Verbose, which shows changed objects (and doesn’t show unchanged objects)
+
+##Modes
+Modes specify the read, write, and execute access, respectively for the three classes of user, group, and others. This is represented as a numerical value, typically either 3 or 4 characters in length. A mode of 755 is typical, and will make a file readable and executable by all three user groups, and writable by the owner only.
```
Comment (jmf-nyu) [inline docs/chmod.md:24]: It would be nice here to include a hyperlink that explains the different numerical values for `chmod`

**Review by jmf-nyu — CHANGES_REQUESTED** [review]
Hi Linda,

Overall, this is a good start! I have suggested a few changes to make sure you are using Markdown syntax properly. I will place the pull request in draft mode so you can make additional changes to your repository and they will be reflected here.

Best,
Jonathan

**Comment by jmf-nyu** [comment]
@ls6758 could you resolve the other changes?

**Comment by ls6758** [comment]
I thought I already did? All of the changes are made on what I can see in my fork of the repository.

File: docs/chmod.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
+#chmod
+##Summary
+The chmod command is used to change the access permissions and special mode flags of files and directories. Only the file owner or a superuser can use the chmod command. 
+
+##Basic command structure 
+$ sudo chmod [options] mode[,mode] file1 [file2…]
```
[the remark this answers is not in the document]
Comment (jmf-nyu) [reply docs/chmod.md:6]: @ls6758 I'm not seeing this formatting correction in your latest commit.

File: docs/chmod.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+```
+-v
+```
+Verbose, which shows changed objects (and doesn’t show unchanged objects)
+
+##Modes
+Modes specify the read, write, and execute access, respectively for the three classes of user, group, and others. This is represented as a numerical value, typically either 3 or 4 characters in length. A mode of 755 is typical, and will make a file readable and executable by all three user groups, and writable by the owner only.
+
+##Output
+This command does not output a message. The ls or stat command would need to be used in order to show that the changes were made and saved correctly. To check if the change of the root folder permissions worked as intended in our Omeka setup, the following command would be run:
```
[the remark this answers is not in the document]
Comment (jmf-nyu) [reply docs/chmod.md:26]: @ls6758 I'm not seeing this formatting change in your latest commit.

File: docs/chmod.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+`-R`
+
+Recursive, which will include objects in subdirectories. This option must be input as a capital R to work as intended.
+
+`-v`
+
+Verbose, which shows changed objects (and doesn’t show unchanged objects)
+
+## Modes
+Modes specify the read, write, and execute access, respectively for the three classes of user, group, and others. This is represented as a numerical value, typically 3 characters in length. A mode of 755 is typical, and will make a file readable and executable by all three user groups, and writable by the owner only. The section entitled "Numeric method" on this web page explains what different modes mean in terms of permissions: https://linuxize.com/post/chmod-command-in-linux/
```
Comment (jmf-nyu) [inline docs/chmod.md:23]: In markdown URLs have the format of `[text for link](Full URL)`. Refer to my `ls` page if needed.

**Comment by jmf-nyu** [comment]
There are still some changes to make, but I'll merge this for now

**Pull request merged**